frame/instrumental-strategy-pablo/Cargo.toml

```diff
@@ -19,12 +19,15 @@ sp-io = { default-features = false, git = "https://github.com/paritytech/substra
 sp-core = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.22" }
 sp-std = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.22" }
 
+composable-maths = { path = "../composable-maths", default-features = false }
+composable-support = { path = "../composable-support", default-features = false }
```

Comment:
After changing of `Cargo.toml` file you also should update `Cargo.lock`. To do this, you can run any `cargo` command (`check`, `build`, `clippy`, etc.), it will update `Cargo.lock` file.